specs/consensus.md

```diff
@@ -66,6 +70,76 @@ Consensus Rules
 
 # Block Validity
 
+## State Transitions
+
+### Validators and Delegations
+
+A transaction `tx` that requests a new validator initializes the [Validator](data_structures.md#validator) field of that account as follows:
+| name              | value                    |
+| ----------------- | ------------------------ |
+| `status`          | `ValidatorStatus.Queued` |
```

Comment:
This Queued status is necessary because we need to track this before the block is finalized for enabling fraud proofs?

Reply:
Cosmos uses:
- `Unbonded -> Bonded -> Unbonding -> Unbonded`
Compared to:
- `Queued -> Bonded -> Unbonding -> Unbonded`

It's just a disambiguation of the first state.

Reply:
Sounds reasonable (and `Queued` is clearer here). But this seems like an optional implementation detail.